docs/develop/python.mdx

```diff
@@ -405,89 +427,3 @@ def foo(ctx, args):
 ### `.options()`
 
 Many of Context's methods support options on the call you are making.
-
```

Comment:
Instead of deleting outright - can we move this content into the drafts - I anticipate that we will recover the batching at some point.

Reply:
Me too! This will always be in the git history when we're ready